client/app/components/button/Review.tsx

```diff
@@ -107,12 +107,12 @@ const Review = ({
   };
 
   const handleChangeRequestConfirm = () => {
-    onRequestChange();
+    onRequestChange?.();
     setShowRequestChangesUndo(true);
   };
 
   const handleUndoRequestChanges = async () => {
-    onUndoRequestChange();
+    onUndoRequestChange?.();
```

Comment:
I haven't used optional chaining to invoke a function that may not exist but it works. Not sure how I feel about it though I used it because our eslint settings wouldn't let me write it how I normally would ie:

`onRequestChange && onRequestChange();`

Reply:
I'm also not v familiar with the `?` usage on a function, but I had a quick look at the docs and you seem to have used it correctly, so I'm happy :)